charts/v1.2/cray-power-control/Chart.yaml

```diff
@@ -1,16 +1,16 @@
 apiVersion: v2
 name: "cray-power-control"
-version: 1.2.3
+version: 1.2.4
 description: "Kubernetes resources for cray-power-control"
 home: "https://github.com/Cray-HPE/hms-power-control-charts"
 sources:
   - "https://github.com/Cray-HPE/hms-power-control"
 dependencies:
   - name: cray-service
-    version: "~9.0.0"
+    version: "~10.0"
     repository: https://artifactory.algol60.net/artifactory/csm-helm-charts
   - name: cray-etcd-base
-    version: "~1.0.0"
+    version: "~1.0"
```

Comment:
My preference would be to stick with listing all three fields in the version, so I don't have to wonder if it means anything special.
"~1.0.0" instead of "~1.0"

However I assume the following have the same meaning: 
">= 1.0.0, < 1.1.0" and ">= 1.0, < 1.1"

https://github.com/Masterminds/semver#tilde-range-comparisons-patch

Reply:
Yeah, this was done based on guidance from the PET team.